scripts/justmat/foulplay.lua

```diff
@@ -59,62 +90,106 @@ local reset = false
 local view = 0     -- 0 == home, 1 == track edit, 2 == global edit 
 local page = 0
 local track_edit = 1
-local stopped = 1 -- change this to 0 if you would like to start running
+local stopped = 1 
+-- added for grid support - junklight 
+local currentmemcell = 1
```

Comment:
a thought for readability would be to make these names consistent w/ your other variables and separate words with `_`s.  for example:

`currentmemcell` => `current_mem_cell`

or, further down:

`gettrack` => `get_track`

obviously this is a taste thing (and i realize the lua libs take a different approach) so of course feel free to ignore!

Reply:
This makes sense. We didn't really talk about variable naming before I merged @junklight's additions. I can give this a go over tomorrow.

Reply:
or not.  honestly it's up to you.  it reads a little clearer to me but it's your call.

Reply:
I think it makes for more easily maintainable code, and probably future mat will thank me for that. :)